tools/defaults.bzl

```diff
@@ -79,6 +95,55 @@ def pkg_npm(name, use_prodmode_output = False, **kwargs):
         deps = deps,
     )
 
+    # Merge package.json with root package.json and perform various substitutions to
+    # prepare it for release. For jq docs, see https://stedolan.github.io/jq/manual/.
+    jq(
+        name = "basic_substitutions",
+        srcs = ["//:package.json", pkg_json],
```

Comment:
nit: mention that order of srcs matters here as well since it is load bearing when passing them to jq